project-base/app/tests/FrontendApiBundle/Functional/Cart/RetrieveCartTest.php

```diff
@@ -442,7 +442,7 @@ private function getExpectedProductDetailWithAllAttributes(): array
                     ],
                     'values' => [
                         [
-                            'text' => t('27"', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $firstDomainLocale),
+                            'text' => t('27', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $firstDomainLocale),
```

Comment:
`'27'` is not a translatable constant anymore, so you can change the to this, right?
```suggestion
                            'text' => '27',
```